README.md

```diff
@@ -324,22 +324,23 @@ aws serverlessrepo put-application-policy \
 
 ### Sceptre
 Create the following [sceptre](https://github.com/Sceptre/sceptre) file
-config/prod/lambda-template.yaml
+config/prod/lambda-finops-cost-rules.yaml
 
 ```yaml
 template:
   type: http
-  url: "https://PUBLISH_BUCKET.s3.amazonaws.com/lambda-template/VERSION/lambda-template.yaml"
+  url: "https://PUBLISH_BUCKET.s3.amazonaws.com/lambda-finops-cost-rules/VERSION/lambda-finops-cost-rules.yaml"
 stack_name: "lambda-template"
 stack_tags:
-  Department: "Platform"
-  Project: "Infrastructure"
   OwnerEmail: "it@sagebase.org"
+parameters:
+  TagList: "CostCenter,CostCenterOther"
+  ChartOfAccountsURL: "https://lambda-mips-api.execute-api.amazonaws.com/accounts"
```

Comment:
if this is sceptre you need to use a resolver.  the problem is that sceptre currently doesn't have a resolver that can do this.  I am planning to create a `UrlGet` resolver for Sceptre which would allow you to do something like `ChartOfAccountsURL: !UrlGet "https://lambda-mips-api.execute-api.amazonaws.com/accounts"`.  You are welcome to participate in the development of that resolver or you can use the [!rcmd](https://docs.sceptre-project.org/latest/docs/resolvers.html#rcmd) resolver as a workaround for  now.